src/test/java/nl/tudelft/pixelperfect/event/FireEventTest.java

```diff
@@ -3,6 +3,7 @@
 /**
  * Created by woute on 5/10/2016.
  */
+@SuppressWarnings("PMD")
```

Comment:
as there very likely will be tests in the future i suggest we dont suppress this warning and keep it as a "reminder"